python_modules/dagster/dagster/core/storage/alembic/versions/011_wipe_schedules_table_for_0_10_0_postgres.py

```diff
@@ -15,12 +16,19 @@
 
 # revision identifiers, used by Alembic.
 revision = "b32a4f3036d2"
-down_revision = "f3e43ff66603"
+down_revision = "375e95bad550"
 branch_labels = None
 depends_on = None
 
 
 def upgrade():
+    bind = op.get_context().bind
+
+    inspector = reflection.Inspector.from_engine(bind)
+
+    if "postgresql" not in inspector.dialect.dialect_description:
+        return
+
```

Comment:
This is a script that shows up in both sqlite/postgres.  This guard is because we do a potentially unknown destructive act below (the `scheduler.wipe` call), so was being extra cautious here and below of only applying it once per dialect.